### DISCUSSION TITLE
Log event before Kubernetes de-dups them

### DISCUSSION THREAD

**Discussion by PakDLiu** [kubernetes-client/csharp#1560] [Q&A]
**Describe the bug**
I am trying to log Kubernetes events as they come in. However, it seems like Kubernetes will correlate the events and if it finds a similar one, it will just increase that event's `count`

Is there anyway to get events as they come in before Kubernetes de-dupes them?
Here is my code right now, but it will only log if there is a **new** event and it doesn't log if the event correlates with an existing event.

```
        var events = client.CoreV1.ListEventForAllNamespacesWithHttpMessagesAsync(watch: true);
        await foreach (var (_, e) in events.WatchAsync<Corev1Event, Corev1EventList>())
        {
           log(e.Message);
        }
```

**Kubernetes C# SDK Client Version**
`13.0.26`

**Server Kubernetes Version**
`1.28.5`

**Dotnet Runtime Version**
N/A

**To Reproduce**
Have Kubernetes correlate the event

**Expected behavior**
A separate log message for every event that comes in

**KubeConfig**
N/A

**Where do you run your app with Kubernetes SDK (please complete the following information):**
 - OS: Linux
 - Environment: container
 - Cloud: Azure

**Additional context**
N/A

**Comment by tg123** [comment]
did you try `EventsV1` corev1event is deprecated

see
https://kubernetes.io/docs/reference/using-api/deprecation-guide/#event-v125

any change should be captured by `watch`

**Comment by PakDLiu** [comment]
That worked, thanks!
